src/DebugConfigurationProvider.ts

```diff
@@ -243,6 +247,8 @@ export class BrightScriptDebugConfigurationProvider implements DebugConfiguratio
         config.cwd = folderUri.fsPath;
         config.rendezvousTracking = config.rendezvousTracking === false ? false : true;
         config.deleteDevChannelBeforeInstall = config.deleteDevChannelBeforeInstall === true;
+        config.remoteControlMode = config.remoteControlMode ? config.remoteControlMode : this.configDefaults.remoteControlMode;
```

Comment:
Let's convert this exclusively to the object here, so downstream we can always just check the `activateOnSessionStart`/`deactivateOnSessionEnd` flags. Something more like this?

```typescript

if (typeof config.remoteControlMode === 'boolean') {
    config.remoteControlMode = {
        activateOnSessionStart: config.remoteControlMode,
        deactivateOnSessionEnd: config.remoteControlMode,
    }
} else {
   config.remoteControlMode = {
        activateOnSessionStart: config.remoteControlMode?.activateOnSessionStart ??  this.configDefaults.remoteControlMode.activateOnSessionStart,
        deactivateOnSessionEnd: config.remoteControlMode?.deactivateOnSessionEnd??  this.configDefaults.remoteControlMode.deactivateOnSessionEnd,
    };
}

```